docs/en_us/developers/source/edx_solutions_api_integration/courses.rst

```diff
@@ -33,20 +33,20 @@ Get a List of Courses
 
 .. code-block:: json
 
-    HTTP 200 OK  
-    Vary: Accept   
-    Content-Type: text/html; charset=utf-8   
-    Allow: GET, HEAD, OPTIONS 
+    HTTP 200 OK
+    Vary: Accept
+    Content-Type: text/html; charset=utf-8
+    Allow: GET, HEAD, OPTIONS
 
     [
         {
-            "category": "course",   
-            "name": "Computer Science 101",   
-            "uri": "http://edx-lms-server/api/courses/un/CS/cs101",   
-            "number": "CS101",   
-            "due": null,   
-            "org": "University N",   
-            "id": "un/CS/cs101"  
+            "category": "course",
```

Comment:
why we have these changes? how are they are related to renaming of api_manager app?